packages/client/ui/react-ui/src/hooks/useCrossmintAuth.ts

```diff
@@ -0,0 +1,12 @@
+import { useContext } from "react";
+import { AuthContext } from "../providers/CrossmintAuthProvider";
+
+export function useCrossmintAuth() {
```

Comment:
This looks like it is always truthy, we shouldn't need to validate on usage.

Reply:
what do you mean? what is always truthy?

Reply:
Nevermind, it was related to the `undefined` question above, but I didn't know about the provider init stuff.